lib/openscap_parser/profile.rb

```diff
@@ -24,13 +24,6 @@ def selected_rule_ids
         @parsed_xml.xpath("select[@selected='true']/@idref").map(&:text)
     end
 
-    def set_values
-      @set_values ||= @parsed_xml.xpath("set-value") &&
-        @parsed_xml.xpath("set-value").map do |set_value|
-        [set_value['idref'], set_value.text]
-      end.to_h
-    end
-
```

Comment:
:+1: it looks like profiles more often use `refine-value` than `set-value` anyway, though all of `select, set-complex-value, set-value, refine-value, refine-rule` are allowed.